osu.Game/Graphics/UserInterface/ExpandingBar.cs

```diff
@@ -13,17 +13,17 @@ namespace osu.Game.Graphics.UserInterface
     /// </summary>
     public class ExpandingBar : Circle
     {
-        private bool isCollapsed;
+        private bool expanded = true;
 
-        public bool IsCollapsed
+        public bool Expanded
         {
-            get => isCollapsed;
+            get => expanded;
             set
             {
-                if (value == isCollapsed)
+                if (value == expanded)
                     return;
 
-                isCollapsed = value;
+                expanded = value;
                 updateState();
             }
         }
```

Comment:
Now that you pointed this out this is basically a poor man's bindable, maybe we want to make it an actual one?

Reply:
still not sure there's much benefit to be had by converting these to bindables, but maybe? @smoogipoo thoughts?

Reply:
Very hard to say in these sort of situations. We've discussed properties vs bindables many times before and the best conclusion was "it's situational". I'm generally fine with leaving properties instead of bindables, unless there's more complex logic such as recursive property setting.

Reply:
Cool, I'll keep that guideline in mind going forward, thanks!